src/Geodesics.jl

```diff
@@ -121,7 +121,7 @@ function forward(lon, lat, azimuth, distance, a, f)::Tuple{Float64,Float64,Float
 
     # Not moving anywhere. We can return the location that was passed in.
     if sigma == 0
-        return phi1, lambda1, alpha12
+        return lambda1, phi1, alpha12
```

Comment:
This is my fault as it's an existing bug, but I see we should really be returning the backazimuth here rather than the azimuth.  Any chance you could add a commit which changes the `alpha12` here to `mod(alpha12 + π, 2π)`?  The test will need to be updated as well.

Reply:
Done